scutout/utils.py

```diff
@@ -775,11 +775,13 @@ def convertImgToJyPixel(cls, filename, outfile, survey=''):
 
         # - MJy/sr units (e.g. HERSCHEL/SPITZER maps)
         elif units == 'MJy/sr':
-            convFactor = 1.e+6*dx*dy/(206265.*206265.)
+            # we need dx and dy in ''
+            convFactor = 1.e+6*(3600*dx)*(3600*dy)/(206265.*206265.)
```

Comment:
For surveys in Jy/beam (see above), dx and dy are used to calculate the conversion factor from the beam area (which is in deg^2), so I think it is better to convert dx and dy to arcsec just here (and explicitly multiplying by 3600, to avoid future issues)

Reply:
Correct, agree with the fix.